lib/mrsk/cli/app.rb

```diff
@@ -219,6 +244,19 @@ def version
     on(MRSK.hosts) { |host| puts_by_host host, capture_with_info(*MRSK.app.current_running_version).strip }
   end
 
+  no_commands do
```

Comment:
imo better just to make this private and then use #send.

Reply:
Changed to `private`